optimum/exporters/ipex/modeling_utils.py

```diff
@@ -732,9 +836,11 @@ def rope(self, query, key, *args, **kwargs):
         return query, key
 
     def postprocess_attention_output(self, attn_output):
+        if self.use_sdpa:
+            attn_output = attn_output.transpose(1, 2).contiguous()
         attn_output = attn_output.reshape(-1, attn_output.shape[-2] * attn_output.shape[-1])
-        attn_output = self.c_proj(attn_output)
-        attn_output = self.resid_dropout(attn_output)
```

Comment:
why is drop removed here?

Reply:
Dropout will be disabled in inference mode, we can do linear add fusion without this dropout.